rest/class-boldgrid-backup-rest-job.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-job.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
+ *
+ * @package    Boldgrid_Backup
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Boldgrid_Backup_Rest_Job
+ *
+ * REST endpoints to perform get jobs.
+ *
+ * @since X.X.X
+ */
+class Boldgrid_Backup_Rest_Job extends Boldgrid_Backup_Rest_Controller {
+
+	/**
+	 * Resource name.
+	 *
+	 * @since  X.X.X
+	 * @access private
+	 * @var    string
+	 */
+	protected $resource = 'jobs';
+
+	/**
+	 * Register the routes for the objects of the controller.
+	 *
+	 * @since X.X.X
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->resource . '/(?P<id>[\w-]+)', [
+			[
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => [ $this, 'get_item' ],
+				'permission_callback' => [ $this, 'permission_check' ],
+				'args'                => [
+					'id' => [
+						'required'    => true,
+						'context'     => [ 'view' ],
+						'description' => esc_html__( 'Unique identifier for the object.', 'boldgrid-backup' ),
+						'type'        => 'string',
+					],
+				],
+			],
+			'schema' => [ $this, 'get_schema' ],
+		] );
+	}
+
+	/**
+	 * Get our sample schema for comments.
+	 *
+	 * @since X.X.X
+	 *
+	 * @return array Schema Format.
+	 */
+	public function get_schema() {
+		$schema = [
```

Comment:
Arrays should be declared with long array syntax per WordPress coding standards
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
See similar comment above.